library/Core.cpp

```diff
@@ -1549,6 +1550,39 @@ bool Core::Init()
     if (!server->listen(RemoteClient::GetDefaultPort()))
         cerr << "TCP listen failed.\n";
 
+    vector<string *> & args = df::global::ui_sidebar_menus->unk.anon_2;
```

Comment:
This should check for the goal being non-null (and the fields should probably be renamed too).